data/json/items/tool_armor.json

```diff
@@ -2882,6 +2916,7 @@
           { "type": "thermo_resin", "covered_by_mat": 100, "thickness": 1.0 }
         ],
         "covers": [ "head" ],
+        "specifically_covers": [ "head_crown", "head_forehead", "head_nape", "head_ear_r", "head_ear_l" ],
```

Comment:
If you do this one you should do the non raised too

Reply:
It's done.